content/api/metrics.apib

```diff
@@ -1348,6 +1348,121 @@ in the world.
+    + domains (optional, list) ... Delimited list of recipient domains to include.
+    + campaigns (optional, list) ... Delimited list of campaigns to include.
+    + templates (optional, list) ... Delimited list of template IDs to include.
+    + sending_ips (optional, list) ... Delimited list of sending IPs to include.
+    + ip_pools (optional, list) ... Delimited list of IP pools to include.
+    + sending_domains (optional, list) ... Delimited list of sending domains to include.
+    + subaccounts (optional, list) ... Delimited list of subaccount IDs to include.
+    + precision = `1min` (optional, enum) ...
+
+        Precision of the time window (`from`, `to`) bounds. All values valid up to a 24 hour window, except for `hour` which is also valid up a 31 day window.
+        See: [Precision Parameter](#header-precision-parameter).
+
+        + Values
+            + `1min`
+            + `5min`
+            + `15min`
+            + `hour`
+
+    + metrics (required, list) ... Delimited list of metrics for filtering.
+
+        + Values
+            + `count_injected`
+            + `count_bounce`
+            + `count_rejected`
+            + `count_delivered`
+            + `count_delivered_first`
+            + `count_delivered_subsequent`
+            + `total_delivery_time_first`
+            + `total_delivery_time_subsequent`
+            + `total_msg_volume`
+            + `count_policy_rejection`
+            + `count_generation_rejection`
+            + `count_generation_failed`
+            + `count_inband_bounce`
+            + `count_outofband_bounce`
+            + `count_soft_bounce`
+            + `count_hard_bounce`
+            + `count_block_bounce`
+            + `count_admin_bounce`
+            + `count_undetermined_bounce`
+            + `count_delayed`
+            + `count_delayed_first`
+            + `count_rendered`
+            + `count_unique_rendered`
+            + `count_unique_confirmed_opened`
+            + `count_clicked`
+            + `count_unique_clicked`
+            + `count_targeted`
+            + `count_sent`
+            + `count_accepted`
+            + `count_spam_complaint`
+
+    + timezone =`UTC` (optional, string) ... Standard timezone identification string.
+    + limit=1000 (optional, int, `5`) ... Maximum number of results to return between 1 and 10000, inclusive.
+    + order_by (optional, string) - Metric by which to order results.
+        + Sample: count_injected
+
++ Request
+
+    + Headers
+
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+            Accept: application/json
+
++ Response 200 (application/json)
+
+        {
+            "results": [
+                {
+                    "count_injected": 109849,
+                    "count_targeted": 110923,
+                    "count_rejected": 1074,
+                    "count_sent": 107610,
+                    "mb_provider": "Verizon Media Group"
```

Comment:
Should we be returning this as `mailbox_provider` to stay consistent with the endpoints and query parameters that we're using?

Reply:
Good Catch, updated!